tests/nl2sql/test_sql_chain.py

```diff
@@ -17,20 +19,15 @@ def test_sql_chain(llm):
     from nl2sql.sql_chain import sql_chain
 
     with patch("nl2sql.sql_chain._create_llm", return_value=llm):
-        answer, sql_query, sql_results = sql_chain("Who are the Swedish drivers?")
+        sql_query, sql_results = sql_chain("Who are the Swedish drivers?")
 
-        assert (
-            answer
-            == "Stefan Johansson, Slim Borgudd, Ronnie Peterson, Gunnar Nilsson, Conny Andersson"
-        )
         assert (
             sql_query
             == 'SELECT "forename", "surname" FROM drivers WHERE "nationality" = \'Swedish\' LIMIT 5;'
         )
-        assert sql_results == [
-            ("Stefan", "Johansson"),
-            ("Slim", "Borgudd"),
-            ("Ronnie", "Peterson"),
-            ("Gunnar", "Nilsson"),
-            ("Conny", "Andersson"),
-        ]
+
+        expected_results = pd.DataFrame({
+            "forename": ["Stefan", "Slim", "Ronnie", "Gunnar", "Conny"],
+            "surname": ["Johansson", "Borgudd", "Peterson", "Nilsson", "Andersson"]
+        })
+        assert_frame_equal(sql_results, expected_results)
```

Comment:
this is really cool. 
You are assuming you know the order of the names returned by the database. You can add `sort_values(["forename", "surname"])`) to both data frames, so even if the order changes the test still passes.